src/Model/Profile.php

```diff
@@ -145,7 +145,7 @@ public static function formatLocation(array $profile)
 	 */
 	public static function load(App $a, $nickname, array $profiledata = [], $show_connect = true)
 	{
-		$user = DBA::selectFirst('user', ['uid'], ['nickname' => $nickname, 'account_removed' => false]);
+		$user = DBA::selectFirst('user', [], ['nickname' => $nickname, 'account_removed' => false]);
```

Comment:
This can be done via the function `User::getByNickname()`.

Reply:
`User::getByNickname()` doesn't seem to filter by `account_removed`. Might that be a problem?

Reply:
I don't think that this is a problem here.

Reply:
Alright, changed.